static/bidder-info/cadent_aperture_mx.yaml

```diff
@@ -0,0 +1,17 @@
+endpoint: "https://hb.emxdgt.com"
+maintainer:
+  email: "contactaperturemx@cadent.tv"
+gvlVendorID: 183
+capabilities:
+  site:
+    mediaTypes:
+      - banner
+      - video
+  app:
+    mediaTypes:
+      - banner
+      - video
+userSync:
+  iframe:
+    url: "https://cs.emxdgt.com/um?ssp=pbs&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect={{.RedirectURL}}"
```

Comment:
We tried validating this user sync URL but it does not seem to be working. Could you please check? Here is the URL that we are trying - 
```
https://cs.emxdgt.com/um?ssp=pbs&gdpr=&gdpr_consent=&us_privacy=&redirect=https%3A%2F%2Fib.adnxs.com%2F%2Fsetuid%3Fbidder%3Dcadent_aperture_mx%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Db%26uid%3D%24UID
```

Reply:
@EMXDigital Requesting to please address this comment.

Reply:
@gargcreation1992 we are looking into this now, will post an update shortly.

Reply:
@gargcreation1992 We've set the endpoint to return a 204 for the time being.  Please let us know if this is sufficient for now to get this code merged in.

Reply:
@EMXDigital We would like to validate the user sync URL before this PR is merged.

Reply:
@gargcreation1992 we have turned the endpoint back on, so kindly notify if any additional issues arise.  Thanks for the patience.

Reply:
@EMXDigital I am still not able to see UID being set after calling the below endpoint - 
```
https://cs.emxdgt.com/um?ssp=pbs&gdpr=&gdpr_consent=&us_privacy=&redirect=http%3A%2F%2Flocalhost%3A8000%2Fsetuid%3Fbidder%3Dcadent_aperture_mx%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Db%26uid%3D%24UID
```

Reply:
@gargcreation1992 When I just hit that same endpoint in a browser, I was 302-redirected to:
`https://localhost:8000/setuid?bidder=cadent_aperture_mx&gdpr=&gdpr_consent=&gpp=&gpp_sid=&f=b&uid=[valid value]`

Looks like it's filling in the UID [valid value] with an actual real value on my end.  What are you seeing happening when you call that endpoint?

Reply:
<img width="1790" alt="Screenshot 2023-08-01 at 12 07 09 PM" src="https://github.com/prebid/prebid-server/assets/8147531/1445a478-054c-453f-9312-15c944a5f843">

I am getting 204. Here is the screenshot for your reference and below is the URL that I am trying to hit - 
```
https://cs.emxdgt.com/um?ssp=pbs&gdpr=&gdpr_consent=&us_privacy=&redirect=http%3A%2F%2Flocalhost%3A8000%2Fsetuid%3Fbidder%3Dcadent_aperture_mx%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Db%26uid%3D%24UID
```

Reply:
Thank you for your patience on this one @EMXDigital! We have tested this on our end and the url is working fine with North American location.